lightspeed/modules/con_lightspeed-process.adoc

```diff
@@ -2,23 +2,23 @@
 
 [id="lightspeed-process_{context}"]
 
-= Using Red Hat Ansible Lightspeed with IBM watsonx Code Assistant
+= Using {LightspeedShortName} with {ibmwatsonxcodeassistant}
 
 .Prerequisites
 
-To use {LightspeedFullName}, ensure that you have the following components:
+To use {LightspeedShortName}, ensure that you have the following components:
 
-* {PlatformNameShort} subscription
+* Trial or paid subscription to {PlatformNameShort}
+* Trial or paid subscription to {ibmwatsonxcodeassistant}
 * VS Code version 1.70.1 or later
 * Ansible extension for VS Code version 2.8 or later
-* {ibmwatsonxcodeassistant} service
 
-== Process for using Red Hat Ansible Lightspeed
+== Process for using {LightspeedShortName}
 
-* Red Hat customer Portal organization administrators can use this guide to: 
+* Red Hat customer portal organization administrators can use this guide to: 
 ** xref:configure-code-assistant_lightspeed-user-guide[Connect {LightspeedShortName} to {ibmwatsonxcodeassistant} instance]
 ** xref:obtain-config-wca-and-model-id_configure-code-assistant[Configure the {ibmwatsonxcodeassistant} model that you want to use]
-** xref:assigning-seat-licenses_lightspeed-user-guide[Assign and remove {LightspeedShortName} seat licenses]
+** xref:configuring-custom-models_lightspeed-user-guide[Configure custom models] 
 * Ansible users (content developers and automators) can use this guide to: 
 ** xref:configuring-with-code-assistant_lightspeed-user-guide[Install and configure Ansible Visual Studio Code extension]
```

Comment:
If possible, I would add a definite article (the) right before Ansible Visual Studio Code extension--so it reads "the ansible visual studio code extension"